#Expyriment 1.0.0 (Python 3.13.7), .xpe-file, coding: cp936
#date: Wed Sep 17 2025 18:58:14
#sha1: 8e8e98
#modules: 
#display: size=(800, 600), window_mode=True, opengl=2
#os: uname_result(system='Windows', node='Shanli', release='11', version='10.0.26100', machine='AMD64')
Time,Type,Event,Value,Detail,Detail2
777,stderr,received,'Exception ignored in: <function Visual.__del__ at 0x000002827AB5CE00>'
778,stderr,received,'Traceback (most recent call last):'
778,stderr,received,'  File "C:\\Users\\23945\\AppData\\Roaming\\Python\\Python313\\site-packages\\expyriment\\stimuli\\_visual.py", line 266, in __del__'
779,stderr,received,'    if self._compression_filename is not None:'
779,stderr,received,"AttributeError: 'Rectangle' object has no attribute '_compression_filename'"
780,stderr,received,'Traceback (most recent call last):'
781,stderr,received,'  File "D:\\UPcite_learning material\\Programming\\Assignments\\Week-2\\Exercises\\square.py", line 21, in <module>\n    rectangle = stimuli.Rectangle(50, 50, colour=(0, 0, 100))'
781,stderr,received,"TypeError: Rectangle.__init__() got multiple values for argument 'colour'"
